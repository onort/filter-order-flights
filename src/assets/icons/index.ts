import Arrival from "./arrival.svg"
import ArrivalDate from "./arrival-date.svg"
import ArrowRight from "./arrow-right.svg"
import Bus from "./bus.svg"
import CheckList from "./checklist.svg"
import Contact from "./contact.svg"
import Departure from "./departure.svg"
import DepartureDate from "./deparutre-date.svg"
import Facebook from "./facebook-icon.svg"
import Forward from "./forward.svg"
import GooglePlus from "./google-plus.svg"
import Help from "./help.svg"
import Home from "./home.svg"
import Location from "./location.svg"
import Luggage from "./luggage.svg"
import Map from "./map.svg"
import Meal from "./meal.svg"
import Menu from "./menu.svg"
import People from "./people.svg"
import Person from "./person.svg"
import Plane from "./plane.svg"
import Sale from "./sale.svg"
import SearchLoction from "./search-location.svg"
import Tick from "./tick.svg"
import Ticket from "./ticket.svg"
import Twitter from "./twitter-icon.svg"

export {
  Arrival,
  ArrivalDate,
  ArrowRight,
  Bus,
  CheckList,
  Contact,
  Departure,
  DepartureDate,
  Facebook,
  Forward,
  GooglePlus,
  Help,
  Home,
  Location,
  Luggage,
  Map,
  Meal,
  Menu,
  People,
  Person,
  Plane,
  Sale,
  SearchLoction,
  Tick,
  Ticket,
  Twitter
}
