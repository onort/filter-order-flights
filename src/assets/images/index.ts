import PegasusLogo from "./pegasus-logo.png"
import ThyLogo from "./thy.png"

export { PegasusLogo, ThyLogo }
