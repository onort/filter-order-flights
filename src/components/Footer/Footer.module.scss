@import "../../styles/main.scss";

.footerContainer {
  display: grid;
  width: 1120px;
  padding: map-get($spacers, x-large) 0;
  margin: map-get($spacers, x-large) auto;
  grid-column-gap: px-to-rem(50px);
  grid-template-columns: 3fr 7fr;
}

.textContent {
  color: $grey2;
  font-family: $font-family-primary;
  font-size: map-get($font-sizes, small);
  font-weight: 400;
  line-height: px-to-rem(24px);
  text-decoration: none;
}

.menu {
  display: grid;
  flex: 5;
  grid-template-columns: repeat(auto-fit, minmax(160px, 1fr));
}
