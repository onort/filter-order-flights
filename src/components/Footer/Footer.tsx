import React from "react"

import styles from "./Footer.module.scss"
import { Container, FooterMenuGroup } from "../"
import { ThyLogo } from "../../assets/images"

export const menuGroupA = {
  links: [
    { text: "Anasayfa", uri: "#" },
    { text: "Şehir & Gezi Rehberi", uri: "#" },
    { text: "Fırsatlar", uri: "#" },
    { text: "Keşfet", uri: "#" },
    { text: "Yardım", uri: "#" },
    { text: "İletişim", uri: "#" }
  ],
  title: "Sayfalar"
}

export const menuGroupB = {
  links: [
    { text: "İstanbul", uri: "#" },
    { text: "Ankara", uri: "#" },
    { text: "İzmir", uri: "#" },
    { text: "New York", uri: "#" },
    { text: "Roma", uri: "#" },
    { text: "Venedik", uri: "#" },
    { text: "Amsterdam", uri: "#" }
  ],
  title: "Popüler Şehirler"
}

export const menuGroupC = {
  links: [
    { text: "Online Check-in", uri: "#" },
    { text: "Genel Uçuş Kuralları", uri: "#" },
    { text: "Yolcu Hakları", uri: "#" }
  ],
  title: "Merak Edilenler"
}

export const menuGroupD = {
  links: [
    { text: "İletişim", uri: "#" },
    { text: "Twitter", uri: "#" },
    { text: "Facebook", uri: "#" }
  ],
  title: "Bize Ulaşın"
}

const Footer: React.SFC = () => {
  return (
    <footer className={styles.footer}>
      <Container className={styles.footerContainer}>
        <div className={styles.branding}>
          <img className={styles.logo} src={ThyLogo} alt="Logo" />
          <p className={styles.textContent}>
            Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec
            viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin
            ipsum quis nunc sollicitudin ultrices.
          </p>
        </div>
        <div className={styles.menu}>
          <FooterMenuGroup title={menuGroupA.title} links={menuGroupA.links} />
          <FooterMenuGroup title={menuGroupB.title} links={menuGroupB.links} />
          <FooterMenuGroup title={menuGroupC.title} links={menuGroupC.links} />
          <FooterMenuGroup title={menuGroupD.title} links={menuGroupD.links} />
        </div>
      </Container>
    </footer>
  )
}

export default Footer
