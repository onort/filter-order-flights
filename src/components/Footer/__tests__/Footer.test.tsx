/* tslint:disable jsx-key */
import React from "react"
import { mount, shallow } from "enzyme"

import { ThyLogo } from "../../../assets/images"
import Footer, {
  menuGroupA,
  menuGroupB,
  menuGroupC,
  menuGroupD
} from "../Footer"
import { FooterMenuGroup } from "../../"

describe("<Footer />", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
    expect(wrapper.html()).toMatchSnapshot()
  })

  it("should have correct children", () => {
    const wrapper = mount(<Footer />)
    expect(
      wrapper.containsAllMatchingElements([
        <img src={ThyLogo} alt="Logo" />,
        <FooterMenuGroup title={menuGroupA.title} links={menuGroupA.links} />,
        <FooterMenuGroup title={menuGroupB.title} links={menuGroupB.links} />,
        <FooterMenuGroup title={menuGroupC.title} links={menuGroupC.links} />,
        <FooterMenuGroup title={menuGroupD.title} links={menuGroupD.links} />
      ])
    )
    wrapper.unmount()
  })
})
