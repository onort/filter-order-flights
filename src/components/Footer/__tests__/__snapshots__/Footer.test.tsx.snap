// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Container
        className="footerContainer"
      >
        <div
          className="branding"
        >
          <img
            alt="Logo"
            className="logo"
            src="thy.png"
          />
          <p
            className="textContent"
          >
            Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.
          </p>
        </div>
        <div
          className="menu"
        >
          <FooterMenuGroup
            links={
              Array [
                Object {
                  "text": "Anasayfa",
                  "uri": "#",
                },
                Object {
                  "text": "Şehir & Gezi Rehberi",
                  "uri": "#",
                },
                Object {
                  "text": "Fırsatlar",
                  "uri": "#",
                },
                Object {
                  "text": "Keşfet",
                  "uri": "#",
                },
                Object {
                  "text": "Yardım",
                  "uri": "#",
                },
                Object {
                  "text": "İletişim",
                  "uri": "#",
                },
              ]
            }
            title="Sayfalar"
          />
          <FooterMenuGroup
            links={
              Array [
                Object {
                  "text": "İstanbul",
                  "uri": "#",
                },
                Object {
                  "text": "Ankara",
                  "uri": "#",
                },
                Object {
                  "text": "İzmir",
                  "uri": "#",
                },
                Object {
                  "text": "New York",
                  "uri": "#",
                },
                Object {
                  "text": "Roma",
                  "uri": "#",
                },
                Object {
                  "text": "Venedik",
                  "uri": "#",
                },
                Object {
                  "text": "Amsterdam",
                  "uri": "#",
                },
              ]
            }
            title="Popüler Şehirler"
          />
          <FooterMenuGroup
            links={
              Array [
                Object {
                  "text": "Online Check-in",
                  "uri": "#",
                },
                Object {
                  "text": "Genel Uçuş Kuralları",
                  "uri": "#",
                },
                Object {
                  "text": "Yolcu Hakları",
                  "uri": "#",
                },
              ]
            }
            title="Merak Edilenler"
          />
          <FooterMenuGroup
            links={
              Array [
                Object {
                  "text": "İletişim",
                  "uri": "#",
                },
                Object {
                  "text": "Twitter",
                  "uri": "#",
                },
                Object {
                  "text": "Facebook",
                  "uri": "#",
                },
              ]
            }
            title="Bize Ulaşın"
          />
        </div>
      </Container>,
      "className": "footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <div
            className="branding"
          >
            <img
              alt="Logo"
              className="logo"
              src="thy.png"
            />
            <p
              className="textContent"
            >
              Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.
            </p>
          </div>,
          <div
            className="menu"
          >
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "Anasayfa",
                    "uri": "#",
                  },
                  Object {
                    "text": "Şehir & Gezi Rehberi",
                    "uri": "#",
                  },
                  Object {
                    "text": "Fırsatlar",
                    "uri": "#",
                  },
                  Object {
                    "text": "Keşfet",
                    "uri": "#",
                  },
                  Object {
                    "text": "Yardım",
                    "uri": "#",
                  },
                  Object {
                    "text": "İletişim",
                    "uri": "#",
                  },
                ]
              }
              title="Sayfalar"
            />
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "İstanbul",
                    "uri": "#",
                  },
                  Object {
                    "text": "Ankara",
                    "uri": "#",
                  },
                  Object {
                    "text": "İzmir",
                    "uri": "#",
                  },
                  Object {
                    "text": "New York",
                    "uri": "#",
                  },
                  Object {
                    "text": "Roma",
                    "uri": "#",
                  },
                  Object {
                    "text": "Venedik",
                    "uri": "#",
                  },
                  Object {
                    "text": "Amsterdam",
                    "uri": "#",
                  },
                ]
              }
              title="Popüler Şehirler"
            />
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "Online Check-in",
                    "uri": "#",
                  },
                  Object {
                    "text": "Genel Uçuş Kuralları",
                    "uri": "#",
                  },
                  Object {
                    "text": "Yolcu Hakları",
                    "uri": "#",
                  },
                ]
              }
              title="Merak Edilenler"
            />
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "İletişim",
                    "uri": "#",
                  },
                  Object {
                    "text": "Twitter",
                    "uri": "#",
                  },
                  Object {
                    "text": "Facebook",
                    "uri": "#",
                  },
                ]
              }
              title="Bize Ulaşın"
            />
          </div>,
        ],
        "className": "footerContainer",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Logo"
                className="logo"
                src="thy.png"
              />,
              <p
                className="textContent"
              >
                Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.
              </p>,
            ],
            "className": "branding",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Logo",
                "className": "logo",
                "src": "thy.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.",
                "className": "textContent",
              },
              "ref": null,
              "rendered": "Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "Anasayfa",
                      "uri": "#",
                    },
                    Object {
                      "text": "Şehir & Gezi Rehberi",
                      "uri": "#",
                    },
                    Object {
                      "text": "Fırsatlar",
                      "uri": "#",
                    },
                    Object {
                      "text": "Keşfet",
                      "uri": "#",
                    },
                    Object {
                      "text": "Yardım",
                      "uri": "#",
                    },
                    Object {
                      "text": "İletişim",
                      "uri": "#",
                    },
                  ]
                }
                title="Sayfalar"
              />,
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "İstanbul",
                      "uri": "#",
                    },
                    Object {
                      "text": "Ankara",
                      "uri": "#",
                    },
                    Object {
                      "text": "İzmir",
                      "uri": "#",
                    },
                    Object {
                      "text": "New York",
                      "uri": "#",
                    },
                    Object {
                      "text": "Roma",
                      "uri": "#",
                    },
                    Object {
                      "text": "Venedik",
                      "uri": "#",
                    },
                    Object {
                      "text": "Amsterdam",
                      "uri": "#",
                    },
                  ]
                }
                title="Popüler Şehirler"
              />,
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "Online Check-in",
                      "uri": "#",
                    },
                    Object {
                      "text": "Genel Uçuş Kuralları",
                      "uri": "#",
                    },
                    Object {
                      "text": "Yolcu Hakları",
                      "uri": "#",
                    },
                  ]
                }
                title="Merak Edilenler"
              />,
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "İletişim",
                      "uri": "#",
                    },
                    Object {
                      "text": "Twitter",
                      "uri": "#",
                    },
                    Object {
                      "text": "Facebook",
                      "uri": "#",
                    },
                  ]
                }
                title="Bize Ulaşın"
              />,
            ],
            "className": "menu",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "links": Array [
                  Object {
                    "text": "Anasayfa",
                    "uri": "#",
                  },
                  Object {
                    "text": "Şehir & Gezi Rehberi",
                    "uri": "#",
                  },
                  Object {
                    "text": "Fırsatlar",
                    "uri": "#",
                  },
                  Object {
                    "text": "Keşfet",
                    "uri": "#",
                  },
                  Object {
                    "text": "Yardım",
                    "uri": "#",
                  },
                  Object {
                    "text": "İletişim",
                    "uri": "#",
                  },
                ],
                "title": "Sayfalar",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "links": Array [
                  Object {
                    "text": "İstanbul",
                    "uri": "#",
                  },
                  Object {
                    "text": "Ankara",
                    "uri": "#",
                  },
                  Object {
                    "text": "İzmir",
                    "uri": "#",
                  },
                  Object {
                    "text": "New York",
                    "uri": "#",
                  },
                  Object {
                    "text": "Roma",
                    "uri": "#",
                  },
                  Object {
                    "text": "Venedik",
                    "uri": "#",
                  },
                  Object {
                    "text": "Amsterdam",
                    "uri": "#",
                  },
                ],
                "title": "Popüler Şehirler",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "links": Array [
                  Object {
                    "text": "Online Check-in",
                    "uri": "#",
                  },
                  Object {
                    "text": "Genel Uçuş Kuralları",
                    "uri": "#",
                  },
                  Object {
                    "text": "Yolcu Hakları",
                    "uri": "#",
                  },
                ],
                "title": "Merak Edilenler",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "links": Array [
                  Object {
                    "text": "İletişim",
                    "uri": "#",
                  },
                  Object {
                    "text": "Twitter",
                    "uri": "#",
                  },
                  Object {
                    "text": "Facebook",
                    "uri": "#",
                  },
                ],
                "title": "Bize Ulaşın",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": [Function],
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Container
          className="footerContainer"
        >
          <div
            className="branding"
          >
            <img
              alt="Logo"
              className="logo"
              src="thy.png"
            />
            <p
              className="textContent"
            >
              Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.
            </p>
          </div>
          <div
            className="menu"
          >
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "Anasayfa",
                    "uri": "#",
                  },
                  Object {
                    "text": "Şehir & Gezi Rehberi",
                    "uri": "#",
                  },
                  Object {
                    "text": "Fırsatlar",
                    "uri": "#",
                  },
                  Object {
                    "text": "Keşfet",
                    "uri": "#",
                  },
                  Object {
                    "text": "Yardım",
                    "uri": "#",
                  },
                  Object {
                    "text": "İletişim",
                    "uri": "#",
                  },
                ]
              }
              title="Sayfalar"
            />
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "İstanbul",
                    "uri": "#",
                  },
                  Object {
                    "text": "Ankara",
                    "uri": "#",
                  },
                  Object {
                    "text": "İzmir",
                    "uri": "#",
                  },
                  Object {
                    "text": "New York",
                    "uri": "#",
                  },
                  Object {
                    "text": "Roma",
                    "uri": "#",
                  },
                  Object {
                    "text": "Venedik",
                    "uri": "#",
                  },
                  Object {
                    "text": "Amsterdam",
                    "uri": "#",
                  },
                ]
              }
              title="Popüler Şehirler"
            />
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "Online Check-in",
                    "uri": "#",
                  },
                  Object {
                    "text": "Genel Uçuş Kuralları",
                    "uri": "#",
                  },
                  Object {
                    "text": "Yolcu Hakları",
                    "uri": "#",
                  },
                ]
              }
              title="Merak Edilenler"
            />
            <FooterMenuGroup
              links={
                Array [
                  Object {
                    "text": "İletişim",
                    "uri": "#",
                  },
                  Object {
                    "text": "Twitter",
                    "uri": "#",
                  },
                  Object {
                    "text": "Facebook",
                    "uri": "#",
                  },
                ]
              }
              title="Bize Ulaşın"
            />
          </div>
        </Container>,
        "className": "footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <div
              className="branding"
            >
              <img
                alt="Logo"
                className="logo"
                src="thy.png"
              />
              <p
                className="textContent"
              >
                Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.
              </p>
            </div>,
            <div
              className="menu"
            >
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "Anasayfa",
                      "uri": "#",
                    },
                    Object {
                      "text": "Şehir & Gezi Rehberi",
                      "uri": "#",
                    },
                    Object {
                      "text": "Fırsatlar",
                      "uri": "#",
                    },
                    Object {
                      "text": "Keşfet",
                      "uri": "#",
                    },
                    Object {
                      "text": "Yardım",
                      "uri": "#",
                    },
                    Object {
                      "text": "İletişim",
                      "uri": "#",
                    },
                  ]
                }
                title="Sayfalar"
              />
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "İstanbul",
                      "uri": "#",
                    },
                    Object {
                      "text": "Ankara",
                      "uri": "#",
                    },
                    Object {
                      "text": "İzmir",
                      "uri": "#",
                    },
                    Object {
                      "text": "New York",
                      "uri": "#",
                    },
                    Object {
                      "text": "Roma",
                      "uri": "#",
                    },
                    Object {
                      "text": "Venedik",
                      "uri": "#",
                    },
                    Object {
                      "text": "Amsterdam",
                      "uri": "#",
                    },
                  ]
                }
                title="Popüler Şehirler"
              />
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "Online Check-in",
                      "uri": "#",
                    },
                    Object {
                      "text": "Genel Uçuş Kuralları",
                      "uri": "#",
                    },
                    Object {
                      "text": "Yolcu Hakları",
                      "uri": "#",
                    },
                  ]
                }
                title="Merak Edilenler"
              />
              <FooterMenuGroup
                links={
                  Array [
                    Object {
                      "text": "İletişim",
                      "uri": "#",
                    },
                    Object {
                      "text": "Twitter",
                      "uri": "#",
                    },
                    Object {
                      "text": "Facebook",
                      "uri": "#",
                    },
                  ]
                }
                title="Bize Ulaşın"
              />
            </div>,
          ],
          "className": "footerContainer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="Logo"
                  className="logo"
                  src="thy.png"
                />,
                <p
                  className="textContent"
                >
                  Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.
                </p>,
              ],
              "className": "branding",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Logo",
                  "className": "logo",
                  "src": "thy.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.",
                  "className": "textContent",
                },
                "ref": null,
                "rendered": "Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <FooterMenuGroup
                  links={
                    Array [
                      Object {
                        "text": "Anasayfa",
                        "uri": "#",
                      },
                      Object {
                        "text": "Şehir & Gezi Rehberi",
                        "uri": "#",
                      },
                      Object {
                        "text": "Fırsatlar",
                        "uri": "#",
                      },
                      Object {
                        "text": "Keşfet",
                        "uri": "#",
                      },
                      Object {
                        "text": "Yardım",
                        "uri": "#",
                      },
                      Object {
                        "text": "İletişim",
                        "uri": "#",
                      },
                    ]
                  }
                  title="Sayfalar"
                />,
                <FooterMenuGroup
                  links={
                    Array [
                      Object {
                        "text": "İstanbul",
                        "uri": "#",
                      },
                      Object {
                        "text": "Ankara",
                        "uri": "#",
                      },
                      Object {
                        "text": "İzmir",
                        "uri": "#",
                      },
                      Object {
                        "text": "New York",
                        "uri": "#",
                      },
                      Object {
                        "text": "Roma",
                        "uri": "#",
                      },
                      Object {
                        "text": "Venedik",
                        "uri": "#",
                      },
                      Object {
                        "text": "Amsterdam",
                        "uri": "#",
                      },
                    ]
                  }
                  title="Popüler Şehirler"
                />,
                <FooterMenuGroup
                  links={
                    Array [
                      Object {
                        "text": "Online Check-in",
                        "uri": "#",
                      },
                      Object {
                        "text": "Genel Uçuş Kuralları",
                        "uri": "#",
                      },
                      Object {
                        "text": "Yolcu Hakları",
                        "uri": "#",
                      },
                    ]
                  }
                  title="Merak Edilenler"
                />,
                <FooterMenuGroup
                  links={
                    Array [
                      Object {
                        "text": "İletişim",
                        "uri": "#",
                      },
                      Object {
                        "text": "Twitter",
                        "uri": "#",
                      },
                      Object {
                        "text": "Facebook",
                        "uri": "#",
                      },
                    ]
                  }
                  title="Bize Ulaşın"
                />,
              ],
              "className": "menu",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "links": Array [
                    Object {
                      "text": "Anasayfa",
                      "uri": "#",
                    },
                    Object {
                      "text": "Şehir & Gezi Rehberi",
                      "uri": "#",
                    },
                    Object {
                      "text": "Fırsatlar",
                      "uri": "#",
                    },
                    Object {
                      "text": "Keşfet",
                      "uri": "#",
                    },
                    Object {
                      "text": "Yardım",
                      "uri": "#",
                    },
                    Object {
                      "text": "İletişim",
                      "uri": "#",
                    },
                  ],
                  "title": "Sayfalar",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "links": Array [
                    Object {
                      "text": "İstanbul",
                      "uri": "#",
                    },
                    Object {
                      "text": "Ankara",
                      "uri": "#",
                    },
                    Object {
                      "text": "İzmir",
                      "uri": "#",
                    },
                    Object {
                      "text": "New York",
                      "uri": "#",
                    },
                    Object {
                      "text": "Roma",
                      "uri": "#",
                    },
                    Object {
                      "text": "Venedik",
                      "uri": "#",
                    },
                    Object {
                      "text": "Amsterdam",
                      "uri": "#",
                    },
                  ],
                  "title": "Popüler Şehirler",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "links": Array [
                    Object {
                      "text": "Online Check-in",
                      "uri": "#",
                    },
                    Object {
                      "text": "Genel Uçuş Kuralları",
                      "uri": "#",
                    },
                    Object {
                      "text": "Yolcu Hakları",
                      "uri": "#",
                    },
                  ],
                  "title": "Merak Edilenler",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "links": Array [
                    Object {
                      "text": "İletişim",
                      "uri": "#",
                    },
                    Object {
                      "text": "Twitter",
                      "uri": "#",
                    },
                    Object {
                      "text": "Facebook",
                      "uri": "#",
                    },
                  ],
                  "title": "Bize Ulaşın",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": [Function],
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`<Footer /> should match snapshot 2`] = `"<footer class=\\"footer\\"><div class=\\"container footerContainer\\"><div class=\\"branding\\"><img class=\\"logo\\" src=\\"thy.png\\" alt=\\"Logo\\"/><p class=\\"textContent\\">Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae ullamcorper metus. Sed sollicitudin ipsum quis nunc sollicitudin ultrices.</p></div><div class=\\"menu\\"><div class=\\"container\\"><h5 class=\\"title\\">Sayfalar</h5><ul class=\\"list\\"><li class=\\"item\\"><a href=\\"#\\">Anasayfa</a></li><li class=\\"item\\"><a href=\\"#\\">Şehir &amp; Gezi Rehberi</a></li><li class=\\"item\\"><a href=\\"#\\">Fırsatlar</a></li><li class=\\"item\\"><a href=\\"#\\">Keşfet</a></li><li class=\\"item\\"><a href=\\"#\\">Yardım</a></li><li class=\\"item\\"><a href=\\"#\\">İletişim</a></li></ul></div><div class=\\"container\\"><h5 class=\\"title\\">Popüler Şehirler</h5><ul class=\\"list\\"><li class=\\"item\\"><a href=\\"#\\">İstanbul</a></li><li class=\\"item\\"><a href=\\"#\\">Ankara</a></li><li class=\\"item\\"><a href=\\"#\\">İzmir</a></li><li class=\\"item\\"><a href=\\"#\\">New York</a></li><li class=\\"item\\"><a href=\\"#\\">Roma</a></li><li class=\\"item\\"><a href=\\"#\\">Venedik</a></li><li class=\\"item\\"><a href=\\"#\\">Amsterdam</a></li></ul></div><div class=\\"container\\"><h5 class=\\"title\\">Merak Edilenler</h5><ul class=\\"list\\"><li class=\\"item\\"><a href=\\"#\\">Online Check-in</a></li><li class=\\"item\\"><a href=\\"#\\">Genel Uçuş Kuralları</a></li><li class=\\"item\\"><a href=\\"#\\">Yolcu Hakları</a></li></ul></div><div class=\\"container\\"><h5 class=\\"title\\">Bize Ulaşın</h5><ul class=\\"list\\"><li class=\\"item\\"><a href=\\"#\\">İletişim</a></li><li class=\\"item\\"><a href=\\"#\\">Twitter</a></li><li class=\\"item\\"><a href=\\"#\\">Facebook</a></li></ul></div></div></div></footer>"`;
